tests/test_models/test_model_process.py

```diff
@@ -0,0 +1,33 @@
+from pydantic import ValidationError
+
+from mio.models.process import DenoiseConfig
+from mio import BASE_DIR
+from mio.utils import hash_video, hash_file
+from ..conftest import DATA_DIR, CONFIG_DIR
+
+def test_interactive_display_config():
+    config = DenoiseConfig.from_id("denoise_test").interactive_display
```

Comment:
@sneakers-the-rat, it might be in docs somewhere, but could you let me know how to find these configurations in the test dir? I'm putting it into the same directory as the test config for stream DAQ, but getting this. Do you need some registering or anything?

>No config with id denoise_test found in /path/to/somewhere/Application Support/mio/config

Reply:
Just leaving a note here so I don't forget I'm skipping these config tests for now.

Reply:
In general I don't think we need these kinda tests (if we wanted to ensure that these values don't change, we could just copy it to tests and assert that the copy in tests was equal to the one in the package), but the fact that it isn't being found would be a bug with the config discovery methods or how we are monkeypatching them to include the `tests/data/config` directory during testing. i'll check this out in the morning